sdk/textanalytics/azure-ai-textanalytics/samples/README.md

```diff
@@ -33,6 +33,7 @@ These sample programs show common scenarios for the Text Analytics client's offe
 |[sample_single_category_classify.py][single_category_classify_sample] and [sample_single_category_classify_async.py][single_category_classify_sample_async]|Use a custom model to classify documents into a single category|
 |[sample_multi_category_classify.py][multi_category_classify_sample] and [sample_multi_category_classify_async.py][multi_category_classify_sample_async]|Use a custom model to classify documents into multiple categories|
 |[sample_model_version.py][sample_model_version] and [sample_model_version_async.py][sample_model_version_async]|Set the model version for pre-built Text Analytics models|
+|[sample_analyze_action_healthcare.py][sample_analyze_action_healthcare] and [sample_analyze_action_healthcare_async.py][sample_analyze_action_healthcare_async]|Run a healthcare and PII analysis together|
```

Comment:
sample_analyze_healthcare_action.py so it sorts next to the other healthcare samples?

Reply:
Done